fleet_adapter_template/config.yaml

```diff
@@ -2,86 +2,81 @@
   account_for_battery_drain: True
   task_capabilities: # Specify the types of RMF Tasks that robots in this fleet are capable of performing
     loop: True
-    delivery: False
+    delivery: True
     clean: False
     finishing_request: "nothing" # [park, charge, nothing]
 
-# DeliveryBot CONFIG =================================================================
+# TinyRobot CONFIG =================================================================
 
 robots:
   # Here the user is expected to append the configuration for each robot in the
   # fleet.
   # Configuration for first robot in this fleet
-  deliverybot1:
+  tinyRobot1:
     robot_config:
-      base_url: "http://192.168.0.1:8888" # API endpoint for the robot
-      user: "some_user"
-      password: "some_password"
-      # max_delay: 10.0 # allowed seconds of delay of the current itinerary before it gets interrupted and replanned
+      max_delay: 15.0 # allowed seconds of delay of the current itinerary before it gets interrupted and replanned
+      filter_waypoints: False
     rmf_config:
       robot_state_update_frequency: 0.5
       start:
         map_name: "L1"
-        # waypoint: "charger_deliverybot1" # Optional
-        # orientation: 0.0 # Optional, radians
-        waypoint: null
-        orientation: null
+        waypoint: "tinyRobot1_charger"
+        orientation: 0.0 # radians
       charger:
-        waypoint: "charger_deliverybot1"
-  # Configuration for the second robot in this fleet if there is a second robot
-  # Uncomment if more than one robot exists.
-  # deliverybot2:
+        waypoint: "tinyRobot1_charger"
+  # Configuration for the second robot in this fleet, uncomment if there is a second robot
+  # tinyRobot2:
   #   robot_config:
-  #     base_url: "http://192.168.0.2:8888" # API endpoint for the robot
-  #     user: "some_user"
-  #     password: "some_password"
-  #     # max_delay: 10.0 # allowed seconds of delay of the current itinerary before it gets interrupted and replanned
+  #     max_delay: 10.0 # allowed seconds of delay of the current itinerary before it gets interrupted and replanned
+  #     filter_waypoints: False
   #   rmf_config:
   #     robot_state_update_frequency: 0.5
   #     start:
   #       map_name: "L1"
-  #       # waypoint: "charger_deliverybot2" # Optional
-  #       # orientation: 0.0 # Optional, radians
-  #       waypoint: null
-  #       orientation: null
+  #       waypoint: "tinyRobot2_charger"
+  #       orientation: 0.0 # radians
   #     charger:
-  #       waypoint: "charger_deliverybot2"
+  #       waypoint: "tinyRobot2_charger"
 
 # TRANSFORM CONFIG =============================================================
 # For computing transforms between Robot and RMF coordinate systems
 
 reference_coordinates:
-  rmf: [[20.33, -3.156],
-        [8.908, -2.57],
-        [13.02, -3.601],
-        [21.93, -4.124]]
-  robot: [[59, 399],
-        [57, 172],
-        [68, 251],
-        [75, 429]]
+  rmf: [[20.42, -5.312],
```

Comment:
Let's keep the original set of waypoints. Having them to be the same might confuse users.